### PR TITLE
Added no sharing criteria

### PR DESCRIPTION
**Pull request by jessemapel**
The linescan and framing models errored when you accessed their parameter sharing criteria. This is not what's suppose to happen when parameters are not sharable. Instead, they should return the default SharingCriteria.

This was causing the default GeometricModel::getParameter and GeometricModel::getParameters to raise an exception when they shouldn't.

### CODE REVIEW

**Review by kberryUSGS — APPROVED** [review]
:+1: This looks good to me!

**Review by scsides — APPROVED** [review]
Approved, I'll let you merge when you want to.

**Pull request merged**
